KPI/E.adoc

```diff
@@ -1,106 +1,147 @@
-     </gmd:maintenanceAndUpdateFrequency>
-    <gmd:userDefinedMaintenanceFrequency>
-      <gts:TM_PeriodDuration>PT6H</gts:TM_PeriodDuration>
-    </gmd:userDefinedMaintenanceFrequency>
-    <gmd:maintenanceNote>
-DRAFT – Guidance on WIS Discovery Metadata (following CBS-16), p38
-    <gco:CharacterString>Instances of bulletin SIKB20NGTT are available every 6 hours starting at 03 UTC.</gco:CharacterString>
-    </gmd:maintenanceNote>
- </gmd:MD_MaintenanceInformation>
-</gmd:resourceMaintenance>
-....
-
-==== E.2.2 What is being measured?
-
-The presence of such element in the WCMP evaluated record.
-
-==== E.2.3 Why should we measure it?
-
-This is an important information which indicates how often the product is updated.
-
-==== E.2.4 How is the measurement executed?
-
-By detecting the presence of such element with a period, and a maintenance note.
-
-See p 37 of the WMO Core Profile Metadata Guidance.
-
-==== E.2.5 What is (are) considered valid results?
-The presence of such element as 
-
-==== E.2.6 Guidance to score well on this assessment
-
-
-==== E.3 Bulletin metadata availability
-
-A Bulletin should contain a temporal extent with begin end and a period for indicating how long the data is kept on the cache (24 hr).
-It should also probably be repeated in the abstract.
-
+. Temporal extent: This element describes the period of time that the available product covers
+. Data Update frequency: The temporal frequency at which the data is updated (i.e. every x hours, days, ...)
+
+=== Status
+
+Draft
+
+=== WCMP Element(s)
+
+* `/gmd:MD_Metadata/gmd:identification//gmd:temporalElement/gmd:EX_TemporalExtent/gmd:extent`
+* `/gmd:MD_Metadata/gmd:identification//gmd:resourceMaintenance//gmd:maintenanceAndUpdateFrequency`
+
+=== What is being measured
+
+The presence of a correctly defined temporal extent and data update frequency.
+
+=== Rationale for measurement
+
+Temporal information is a significant characteristic of weather/climate/water
+data and as such is critical for users to know which period(s) of time is/are
+covered by products and how often new products are received.
+
+=== Rules for implementation
+
+The TemporalExtent is defined in `gmd:temporalElement`:
+
+```xml
+<gmd:identificationInfo>
+  ...
+  <gmd:temporalElement>
+    <gmd:EX_TemporalExtent id="boundingTemporalExtent">
+      <gmd:extent>
+        <gml:TimePeriod gml:id="boundingTemporalExtentPeriod">
+          <gml:beginPosition>2005-10-01</gml:beginPosition>
+          <gml:endPosition>2014-10-20</gml:endPosition>
+        </gml:TimePeriod>
+      </gmd:extent>
+    </gmd:EX_TemporalExtent>
+  </gmd:temporalElement>
+  ...
+</gmd:identificationInfo>
+```
+
+In the case of an ongoing dataset with a known start date, but no known end
+date can be defined, the end date much be defined as `now`.  For instance,
```

Comment:
There seems to be a typo "the end date **much** be defined ", I am not sure if you meant "must" or "may" :-)

Reply:
yep thanks. done